cmake/ccache.cmake

```diff
@@ -1,7 +0,0 @@
-if (CMAKE_VERSION VERSION_LESS "3.17")
```

Comment:
Please restore this file and change the version to check for to `3.0`.

Reply:
Done.